cypress/support/commands.js

```diff
@@ -32,7 +32,8 @@ Cypress.Commands.add('getByDataCy', (selector) => {
   cy.get(`[data-cy="${selector}"]`);
 });
 
-Cypress.Commands.add('register', (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
+Cypress.Commands.add('register', (email = 'riot@qa.team', username = 'riot',
+  password = '12345Qwert!') => {
```

Comment:
The `register` command uses default values for email, username, and password. While this is convenient, it might lead to conflicts if multiple tests run concurrently or if the same credentials are used across different tests. Consider using dynamic values or the `user` object generated in the tests to ensure unique credentials for each test run.